brion/spikeReport.cpp

```diff
@@ -60,13 +44,31 @@ class SpikeReport
         plugin.reset( SpikePluginFactory::getInstance().create( initData ));
     }
 
-    boost::scoped_ptr< SpikeReportPlugin > plugin;
+public:
+    std::unique_ptr< SpikeReportPlugin > plugin;
+    lunchbox::ThreadPool threadPool {1};
```

Comment:
std::async will start a new thread for each call. The thread pool reuses the same threads